cmd/ui.go

```diff
@@ -167,16 +180,22 @@ func printExecutionDescription(
 		scenarioDesc = fmt.Sprintf("%d scenarios", len(executorConfigs))
 	}
 
-	fprintf(gs.stdOut, "  scenarios: %s\n", valueColor.Sprintf(
+	fmt.Fprintf(buf, "  scenarios: %s\n", valueColor.Sprintf(
 		"(%.2f%%) %s, %d max VUs, %s max duration (incl. graceful stop):",
 		conf.ExecutionSegment.FloatLength()*100, scenarioDesc,
 		lib.GetMaxPossibleVUs(execPlan), maxDuration.Round(100*time.Millisecond)),
 	)
 	for _, ec := range executorConfigs {
-		fprintf(gs.stdOut, "           * %s: %s\n",
+		fmt.Fprintf(buf, "           * %s: %s\n",
 			ec.GetName(), ec.GetDescription(et))
 	}
-	fprintf(gs.stdOut, "\n")
+	fmt.Fprintf(buf, "\n")
+
+	if gs.flags.quiet {
+		gs.logger.Debug(buf.String())
```

Comment:
Why are we logging to debug though :thinking: ?

Reply:
My logic here is that we should try to not spam `INFO` log-level messages for k6 internals, we should leave this level for users if at all possible. k6 should still emit `ERROR` and `WARNING` messages, of course, but everything else should be pretty much be `DEBUG` if possible :man_shrugging: